Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM library/node:6.3
+
+WORKDIR /src
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+RUN node_modules/bower/bin/bower --allow-root install \
+	&& tools/prepare.sh
+
+EXPOSE 3000
+
+ENTRYPOINT ["/usr/local/bin/npm"]
```

Comment:
Setting the entrypoint to npm allows you to do things like `docker run resin/docs test` and have it be the equivalent of running `npm test` inside the container. That's sort of a contrived example, but you can do things like `start --production` to switch environments, etc.